endpoints/info/bidders_test.go

```diff
@@ -175,88 +152,60 @@ func TestPrepareBiddersResponseEnabledOnly(t *testing.T) {
+			name:         "none",
+			givenBidders: config.BidderInfos{},
+			expected:     `[]`,
 		},
 		{
-			name:                "core-one-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledCore, "a": enabledCore},
-			givenRequestAliases: nil,
-			expected:            `["a","z"]`,
+			name:         "core-one-enabled",
+			givenBidders: config.BidderInfos{"a": enabledCore},
+			expected:     `["a"]`,
 		},
 		{
-			name:                "alias-one",
-			givenBidders:        config.BidderInfos{"a": enabledAlias},
-			givenRequestAliases: nil,
-			expected:            `["a"]`,
+			name:         "core-one-disabled",
+			givenBidders: config.BidderInfos{"a": disabledCore},
+			expected:     `[]`,
 		},
 		{
-			name:                "alias-mixed",
-			givenBidders:        config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
-			givenRequestAliases: nil,
-			expected:            `["c","d"]`,
+			name:         "core-one-mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": enabledCore},
+			expected:     `["b"]`,
 		},
 		{
-			name:                "alias-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledAlias, "a": enabledCore},
-			givenRequestAliases: nil,
-			expected:            `["a","z"]`,
+			name:         "core-one-mixed-sorted",
+			givenBidders: config.BidderInfos{"z": enabledCore, "a": enabledCore},
+			expected:     `["a","z"]`,
 		},
 		{
-			name:                "defaultrequest-one",
-			givenBidders:        config.BidderInfos{"a": enabledCore},
-			givenRequestAliases: map[string]string{"b": "a"},
-			expected:            `["a","b"]`,
+			name:         "alias-one",
+			givenBidders: config.BidderInfos{"a": enabledAlias},
+			expected:     `["a"]`,
 		},
 		{
-			name:                "defaultrequest-mixed",
-			givenBidders:        config.BidderInfos{"a": enabledCore, "b": disabledCore},
-			givenRequestAliases: map[string]string{"x": "a", "y": "b"},
-			expected:            `["a","x"]`,
+			name:         "alias-mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["c","d"]`,
 		},
 		{
-			name:                "defaultrequest-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledCore},
-			givenRequestAliases: map[string]string{"a": "z"},
-			expected:            `["a","z"]`,
+			name:         "alias-mixed-sorted",
+			givenBidders: config.BidderInfos{"z": enabledAlias, "a": enabledCore},
+			expected:     `["a","z"]`,
 		},
 		{
-			name:                "mixed",
-			givenBidders:        config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
-			givenRequestAliases: map[string]string{"z": "a"},
-			expected:            `["c","d"]`,
+			name:         "mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["c","d"]`,
```

Comment:
I think you can delete this test case. It is the same as the `alias-mixed` case on lines 190-192.

Reply:
Differentiated.